rapidast.py

```diff
@@ -125,6 +129,48 @@ def run_scanner(name, config, args, scan_exporter):
     return 0
 
 
+def dump_redacted_config(config_file_location: str, destination_dir: str) -> bool:
+    """
+    Redacts sensitive parameters from a configuration file and writes the redacted
+    version to a destination directory
+
+    Args:
+        config_file_location: The file path to the source configuration file
+        destination_dir: The directory where the redacted configuration file should be saved
+
+    """
+    logging.info("Starting the redaction and dumping process for the configuration files")
+
+    try:
+        if not os.path.exists(destination_dir):
+            os.makedirs(destination_dir)
+            logging.info(f"Created destination directory: {destination_dir}")
+
+        if os.path.exists(DEFAULT_CONFIG_FILE):
+            logging.info(f"Copying default config file from {DEFAULT_CONFIG_FILE} to {destination_dir}")
+            shutil.copy(DEFAULT_CONFIG_FILE, destination_dir)
```

Comment:
it would be a very rare case, but not impossible for a user to include the authentication config into rapidast-defaults.yaml and use it. Can we also redact the info if it exists in the same way?

Reply:
Done